.github/workflows/eslint.yml

```diff
@@ -0,0 +1,88 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# ESLint is a tool for identifying and reporting on patterns
+# found in ECMAScript/JavaScript code.
+# More details at https://github.com/eslint/eslint
+# and https://eslint.org
+
+name: ESLint
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main" ]
+  schedule:
+    - cron: '24 21 * * 4'
+
+jobs:
+  eslint_client:
+    name: client eslint
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      security-events: write
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install ESLint
+        working-directory: ./client
+        run: |
+          npm install eslint@8.56.0
+          npm install @microsoft/eslint-formatter-sarif@2.1.7
+          npm install eslint-plugin-react@latest --save-dev
+          npm install @typescript-eslint/eslint-plugin@latest --save-dev --force
+
+      - name: Run ESLint
+        working-directory: ./client
+        run: npx eslint .
+          --config .eslintrc.cjs
+          --ext .js,.jsx,.ts,.tsx
+          --format @microsoft/eslint-formatter-sarif
+          --output-file eslint-results.sarif
+        continue-on-error: true
+
+      - name: Upload analysis results to GitHub
+        uses: github/codeql-action/upload-sarif@v3
+        with:
+          sarif_file: ./client/eslint-results.sarif
+          wait-for-processing: true
+
+  eslint_server:
+    name: server eslint
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      security-events: write
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install ESLint
+        working-directory: ./server
+        run: |
+          npm install eslint@8.56.0
+          npm install @microsoft/eslint-formatter-sarif@2.1.7
+          npm install eslint-plugin-react@latest --save-dev
+
+      - name: Run ESLint
+        working-directory: ./server
+        run: npx eslint .
+          --config .eslintrc.json
+          --ext .js,.jsx,.ts,.tsx
+          --format @microsoft/eslint-formatter-sarif
+          --output-file eslint-results.sarif
+        continue-on-error: true
```

Comment:
Same with this continue-on-error, it should be removed

Reply:
removed